setup.py

```diff
@@ -1668,6 +1668,20 @@ class db_found(Exception): pass
         if '_tkinter' not in [e.name for e in self.extensions]:
             missing.append('_tkinter')
 
+        # Build the _uuid module if possible
+        build_uuid = False
+        if find_file("uuid.h", inc_dirs, ["/usr/include/uuid"]):
```

Comment:
Since you add /usr/include/uuid to inc_dirs, I would maybe store find_file() result into uuid_incdirs and then pass it to Extension(): inc_dirs=uuid_incdirs? Similar code is used for ssl_incs.

Reply:
Ok.